lib/xapi/track/image.rb

```diff
@@ -0,0 +1,18 @@
+module Xapi
+  # Image is a image associated with documentation of specific language
+  class Image
+    attr_reader :path
+    def initialize(image_name, path)
+      @image_name = image_name
```

Comment:
Not a big deal here, but usually if you're inside of a class named `Image` there's no need to prefix anything with `image` on the inside, so I would probably have named this just `name`.